spec/index.html

```diff
@@ -273,17 +273,19 @@ <h3>Serializing the Results Table</h3>
       <h3>Serializing RDF Terms</h3>
       <p>The SPARQL Results TSV Results Format serializes RDF terms in the results table by using the syntax that SPARQL
        [<a href="#SPARQL11-QUERY">SPARQL11-QUERY</a>] and Turtle [<cite><a class="bibref" rel="biblioentry" href="#bib-TURTLE">TURTLE</a></cite>] use.</p>
-      <p>IRIs enclosed in <code>/&lt;...&gt;</code>, literals are enclosed with double quotes <code>/"</code>...<code>/"</code> or single quotes <code>/'</code> ...<code>/'</code> with optional <code>/@lang</code> or <code>/^^</code>
-      for datatype. The quotes around the lexical form is required. Tab, newline and carriage return characters (Unicode codepoints 0x09, 0x0A (line feed) and 0x0D (Carriage Return)) are encoded in
-      strings as <code>/\t</code>, <code>/\n</code> and <code>/\r</code> respectively. The long string forms using triple quotes <code>/"""</code> and <code>/'''</code> <em class="rfc2119" title="must not">must not</em> be
+      <p>IRIs are enclosed in <code>&lt;...&gt;</code>, literals are enclosed with double quotes <code>"</code>...<code>"</code> or single quotes <code>'</code> ...<code>'</code> with optional <code>@lang</code> or <code>^^</code>
+      for datatype.
+      IRIs <em class="rfc2119" title="must">must</em> be absolute, and datatypes <em class="rfc2119" title="must">must</em> be written without prefixes in their absolute form enclosed in <code>&lt;...&gt;</code>.
```

Comment:
RFC 3986 terminolgy is strange. There isn't a clear piece of language.

"absolute", as in the rule `absolute-URI`, means there isn't a fragment.

"IRI" is the correct rule from the grammar from the RFC but we all use that loosely.

```suggestion
      IRIs are written enclosed in <code>&lt;...&gt;</code>.
      They <em class="rfc2119" title="must">must</em> conform to the <a href="https://www.rfc-editor.org/rfc/rfc3987#section-2.2">IRI rule</a> of [[[RFC3987]]].
      Such IRIs include the IRI scheme and <em class="rfc2119" title="must not">must not</em>be <a href="https://www.rfc-editor.org/rfc/rfc3986#section-4.1>IRI references</a>.
      This includes IRIs used as datatypes.
```

Reply:
> Such IRIs include the IRI scheme and <em class="rfc2119" title="must not">must not</em>be <a href="https://www.rfc-editor.org/rfc/rfc3986#section-4.1>IRI references</a>

This refers to URI references, with can either be an URI or a relative-ref.
But the URI option _is_ allowed, so this might be a bit confusing.
Should we explicitly refer to relative-ref's that are not allowed? (https://www.rfc-editor.org/rfc/rfc3986#section-4.2)

Reply:
Good point.

Switched to linking to section 4.2 :

```suggestion
      IRIs are written enclosed in <code>&lt;...&gt;</code>.
      They <em class="rfc2119" title="must">must</em> conform to the <a href="https://www.rfc-editor.org/rfc/rfc3987#section-2.2">IRI rule</a> of [[[RFC3987]]].
      Such IRIs include the IRI scheme and <em class="rfc2119" title="must not">must not</em> be a <a href="https://www.rfc-editor.org/rfc/rfc3986#section-4.2>Relative Reference</a>.
      This includes IRIs used as datatypes.
```